src/components/RecipientChartFilters.js

```diff
@@ -38,6 +39,7 @@ const RecipientChartFilters = (props) => {
         }}
       />
       <Select
+        classPrefix="recipient-chart-breakdown"
```

Comment:
Unless you're adding a new class, please use the existing React Select class names ... so, `classNamePrefix` ... results in less changes.